core/runtime/src/main/java/io/quarkus/runtime/Timing.java

```diff
@@ -50,12 +50,13 @@ public static void restart() {
         bootStartTime = System.nanoTime();
     }
 
-    public static void printStartupTime(String version, String features) {
+    public static void printStartupTime(String version, String features, String profile, boolean liveCoding) {
         final long bootTimeNanoSeconds = System.nanoTime() - bootStartTime;
         final Logger logger = Logger.getLogger("io.quarkus");
         //Use a BigDecimal so we can render in seconds with 3 digits precision, as requested:
         final BigDecimal secondsRepresentation = convertToBigDecimalSeconds(bootTimeNanoSeconds);
         logger.infof("Quarkus %s started in %ss. %s", version, secondsRepresentation, httpServerInfo);
+        logger.infof("Profile %s activated. %s", profile, liveCoding ? "Live Coding activated." : "");
```

Comment:
What is "Live Coding"? Is that the dev mode when `quarkus:dev` is used?

Reply:
It should be "Live Reload" : https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/0-glossary.adoc

Reply:
We switched to Live Coding as a term a while back. I guess we have not propagated it. But it was for the general experience and maybe live reload is better in this case as it's specifically the live reload option. Not sure.

Reply:
IMHO "Live coding" refers to performing programmation in front of people (like in conferences).
https://en.wikipedia.org/wiki/Live_coding

Here we have a automatic reload (or hot reload like in hot code replacement)

Reply:
>Here we have a automatic reload (or hot reload like in hot code replacement)

Right, my only concern is that it may confuse users since the reload happens when you actually perform a request to the application and not automatically.

Reply:
For the sake of not going over for ever. I'd propose to push that PR in and if people really hate Live Coding, we can always easily rename later. It's not like it's an API.

Reply:
> Well @loicmathieu The Wikipedia Live Coding definition is pretty spot on for us
> 
> > Live coding, [...], makes programming an integral part of the running program.

Yeah, I add it on purpose, it says both usage, I particularly love the reference to "performing art"
> It is most prominent as a performing arts form